app/models/articles.model.js

```diff
@@ -49,3 +48,26 @@ ORDER BY
     return result.rows;
   });
 };
+
+exports.selectCommentById = (article_id) => {
+  return db
+    .query(
+      `
+SELECT
+ comment_id, votes, created_at, author, body, article_id
+FROM 
+ comments
+WHERE
+ article_id = $1
+ORDER BY
+ created_at DESC;
+`,
+      [article_id]
+    )
+    .then((result) => {
+        if (result.rows.length === 0){
```

Comment:
If I write a brand new article, no one would have commented on it yet. Would that be an error?

Have a think about the difference between an article not existing and comments not existing. Refer to the advanced error handling notes for help.